### DISCUSSION TITLE
Your favorite TV channels

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#293] [General]
Please share guys if you still watch TV and what channels mostly.

**Comment by MirkosMangle** [comment]
ESPN, BBC, Fubo TV. I watch only sport channels.

**Reply by XelaAl** [reply]
> ESPN, BBC, Fubo TV. I watch only sport channels.

I also watch mostly fox sports. It offers everything I need. Live scores, odds, team news, player news, streams, videos, stats, standings & schedules covering NFL, MLB, NASCAR, WWE, NBA, NHL, etc. And the fox sports customer service works well https://fox-sports.pissedconsumer.com/customer-service.html. It also has to be noted, the app is free to download.

**Comment by ellielo** [comment]
How much do you pay for FUBO monthly? I wanted to make a subscription too, but the prices where a bit too expensive for me. I have some free sport streaming sites on my firestick [https://www.firesticktricks.com/free-sports-streaming-sites.html](url) and it actually is enough.
